docs/packaging/updating-an-existing-package.md

```diff
@@ -36,10 +94,63 @@ Example:
 /usr/share/ypkg/yupdate.py 1.0 https://example.com/example-1.0.tar.xz
 ```
 
-## Building a package
+## The `MAINTAINERS.md` File
 
-After bumping or updating the package, build it:
+There must be a file called `MAINTAINERS.md` using the template in [Maintainership](procedures/maintainership.md). Add it if it does not already exist. It should name the current maintainer(s) of the package.
+
+## Build the package
+
+After bumping or updating the package, build it by running `make`.
+Once your package has built successfully, you will need to [test it](testing-a-package).
+
+## Commit Your Changes
+
+Check the [changes in your files](git-basics).
+
+[Add / remove files as necessary to the commit](git-basics.md). Then, **check your branch**.
+
+Run `git status`. Make sure all the files you changed are staged, and that there are no untracked files. When all it well, run `git commit`.
+
+### Commit message format for updated / bumped packages
+
+There should be a meaningful summary line (with the package name), a blank line, and then the rest of the commit message.
+
+- Bullet point lists should start with a dash.
+- Include a changelog with a brief list of updates from the upstream release notes, with no links.
+- There may also be a section for Solus specific work (e.g. rebuild against x / rework to remove dependency).
+- Optional: A link to the upstream release notes page.
+- Include your Test Plan.
+
+Here is an example in our standard format:
 
-```bash
-make
 ```
+Update foo to 1.2.3
+
+**Changelog**
```

Comment:
We had chosen asterisk bolding here because hash bolding would require special git flags. Otherwise on command line, git commit strips the hashes out, treating them as comments . I've recommend updating the PR template in Silke's monorepo PR https://github.com/getsolus/packages/pull/344
We might decide on a different heading marker (like =) or work around the git issue.